internal/app/commands/sandbox/distro.go

```diff
@@ -26,7 +26,7 @@ const DISTRO_FOLDER_NAME_TPL = "enonic-xp-%s-sdk-%s"
 // old CLI used this format to store distro name in sandbox, new format will apply after first sandbox distro change
 const OLD_DISTRO_FOLDER_NAME_REGEXP = "^(?:windows|mac|linux)-(?:sdk|server)-([-_.a-zA-Z0-9]+)$"
 
-const DISTRO_LIST_NAME_REGEXP = "^(?:windows|mac|mac-arm64|linux|linux-arm64)-(?:sdk|server)-([-_.a-zA-Z0-9]+) \\([ ,a-zA-Z]+\\)$"
+const DISTRO_LIST_NAME_REGEXP = "^(?:windows|mac|mac-arm64|linux|linux-arm64)-(?:sdk|server)-([-_.a-zA-Z0-9]+) (?:\\([ ,a-zA-Z]+\\))?$"
```

Comment:
This regexp expects space at the end:
`mac-sdk-2.2`  <-- won't be caught
`mac-sdk-2.2 ` <-- will be caught
And looking at the template we use it will work fine:
`const DISTRO_LIST_NAME_TPL = "%s-sdk-%s %s"`
So this is not breaking but I would make trailing space optional for future proofing.

Reply:
Yeah, because of how we construct that string, trailling space is always present ATM. But I agree it can break things in the future, if there are more changes to come.
I'll make it optional. I'd even suggest to add tests for those utility functions (not in this PR though).